test/gui/shared/scripts/bdd_hooks.py

```diff
@@ -48,6 +51,8 @@ def hook(context):
         context.userData['localBackendUrl'] = 'https://localhost:9200'
     if context.userData['clientSyncPath'] == '':
         context.userData['clientSyncPath'] = '/tmp/client-bdd/'
+    if context.userData['clientSyncPathUser'] == '':
+        context.userData['clientSyncPath'] = '/tmp/client-bdd-user/'
```

Comment:
I don't understand what that will help, it will still be the same folder that will be used in different tests.
Also it will be very confusing `clientSyncPath` or `clientSyncPathUser` 
Also line 54 is using `clientSyncPathUser` but 55 `clientSyncPath`
can we maybe have the user-name in the path name?

Reply:
Currently different user should have different sync path. So, for 2nd user we need a new `clientSyncPath` for now I have called it `clientSyncPathUser2` so that it will be less confusing.